### PR TITLE
[stable/coredns] allow nodeselector configuration

### PR DESCRIPTION
**Pull request by crandles**
@andor44 @shashidharatd

### CODE REVIEW

File: stable/coredns/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: coredns
-version: 1.0.1
+version: 1.0.2
```
Comment (davidkarlsen) [inline stable/coredns/Chart.yaml]: make it 1.1.0 as it is a new feature

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/coredns/templates/deployment.yaml [hunk 2]
```diff
@@ -49,6 +49,10 @@ spec:
       {{- if .Values.tolerations }}
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
+      {{- end }}
+      {{- if .Values.nodeSelector }}
```
Comment (davidkarlsen) [inline stable/coredns/templates/deployment.yaml:53]: please also add a default to values.yaml (empty) and document in readme.

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/coredns/templates/deployment.yaml (hunk 2 above)
Comment (davidkarlsen) [reply stable/coredns/templates/deployment.yaml:53]: This is a good example https://github.com/helm/charts/pull/7777/files

**Comment by crandles** [comment]
Done. Thanks!

File: stable/coredns/templates/deployment.yaml (hunk 2 above)
Comment (davidkarlsen) [reply stable/coredns/templates/deployment.yaml:53]: Still can't see anything in the Readme?

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/coredns/templates/deployment.yaml (hunk 2 above)
Comment (crandles) [reply stable/coredns/templates/deployment.yaml:53]: This chart doesn't have any of the variables described in the README

Comment (crandles) [reply stable/coredns/templates/deployment.yaml:53]: Happy to add, but I think that might be worth following up in another PR, as that would cover more than I am trying to add here.

Comment (davidkarlsen) [reply stable/coredns/templates/deployment.yaml:53]: ah - so it does not - ok

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**